keg_elements/forms/__init__.py

```diff
@@ -200,6 +201,26 @@ def selected_choice_label(self):
         return value_dict.get(self.data)
 
 
+class RequiredRadioBooleanField(wtforms.fields.RadioField):
+    def __init__(self, *args, **kwargs):
+        yes_label = kwargs.pop('yes_label', 'Yes')
+        no_label = kwargs.pop('no_label', 'No')
+
+        def bool_coerce(val):
+            if val == u'True':
+                return True
+            if val == u'False':
+                return False
+            return val
+
+        kwargs['choices'] = [(True, yes_label), (False, no_label)]
+        kwargs['coerce'] = bool_coerce
+        kwargs['validators'] = [InputRequired()]
```

Comment:
What if my DB backend had bit fields (mssql) that I wanted to use this with?  So the values actually need to be 0 or 1.

Validation: I could want to add a custom validator that makes sure that verifies a value on a different field.  If that field is X, this field can't be Y, etc.

The point is, the kwargs are there for a reason.  Why prevent them from being used when the code to add to them is almost as easy and leaves the API the most flexible?